### DISCUSSION TITLE
Why Not Add References to Readme Files?

### DISCUSSION THREAD

**Discussion by mgifford** [retextjs/retext#64] [Ideas] [answered]
I'd love to have references here to the sources that you and your team are drawing on for this. Particularly for the equality library.

**Comment by wooorm** [comment, marked as answer]
What types of references are you looking for? Also: there is no team. This is mostly stuff I’ve created over the years and that folks have contributed to.

For retext-equality specifically, several of the rules have a `source` field. But this wasn’t initially there, some of that is lost.

**Reply by mgifford** [reply]
I was just looking for links of organizations who are avoiding or managing lists like the one that you maintain here. This evolves over time, so would be great to highlight what sources have been looked at.  

I know Twitter, MySQL & other are updating their language, but useful to know who the leaders are in this space and who you are following.

Thanks @wooorm for your work on this. Quite the interesting language for improving accessibility.

**Reply by wooorm** [reply]
Some of those can be found in the source of retext-equality indeed. Others are added by the community, typically based on such sources, which can be seen by browsing the issues and PRs.
